api/domain/projections/certificate.proto

```diff
@@ -18,6 +18,7 @@ syntax = "proto3";
 // https://cloud.google.com/apis/design/naming_convention
 
 import "api/domain/projections/metadata.proto";
+import "validate/validate.proto";
```

Comment:
Projections do not need to be validated IMHO. They are always right since they are the internal state of the m8 control plane.